nvflare/fuel/f3/drivers/aio_grpc_driver.py

```diff
@@ -251,7 +254,10 @@ def __init__(self):
 
     @staticmethod
     def supported_transports() -> List[str]:
-        return ["grpc", "grpcs"]
+        if use_aio_grpc():
+            return ["grpc", "grpcs"]
+        else:
+            return ["agrpc", "agrpcs"]
```

Comment:
Should it be "grpc" VS "nagrpc"?
"grpcs" VS "nagrpcs"?

Reply:
See my answers above.